source/deploy/try-mattermost.rst

```diff
@@ -0,0 +1,80 @@
+Try Mattermost
```

Comment:
```suggestion
Evaluation Deployments
```

How does this resonate? When I think of "Try Mattermost", there is more information that I want to get from the doc, including how to unlock advanced features and what successful a evaluation looks like. I think that page lives in another place accessible to more personas who might not directly resonate with the "deploy".